Assets/Scripts/GEGIntensityManager.cs

```diff
@@ -211,20 +217,20 @@ void HighM()
             EnemyPropertyChange("ZomBearHealth", 0, 150);
             EnemyPropertyChange("ZomBearSpeed", 0, 8);
             EnemyPropertyChange("ZomBearAttackDamage", 0, 30);
-            EnemyPropertyChange("ZomBearAttackRate", 0, (float)1);
+            EnemyPropertyChange("ZomBearAttackRate", 0, 1f);
             EnemyPropertyChange("ZomBearHealth", 1, 120);
             EnemyPropertyChange("ZomBearSpeed", 1, 8);
             EnemyPropertyChange("ZomBearAttackDamage", 1, 25);
-            EnemyPropertyChange("ZomBearAttackRate", 1, (float)1.5);
+            EnemyPropertyChange("ZomBearAttackRate", 1, 1.5f);
             EnemyPropertyChange("ZomBearHealth", 2, 150);
             EnemyPropertyChange("ZomBearSpeed", 2, 15);
             EnemyPropertyChange("ZomBearAttackDamage", 2, 30);
-            EnemyPropertyChange("ZomBearAttackRate", 2, (float)2);
+            EnemyPropertyChange("ZomBearAttackRate", 2, 2f);
         }
 
         public void IntensityIncrease(float i)
         {
-            intensity += i;
+            intensity = Mathf.Clamp(intensity + i, 0, 100);
```

Comment:
using Clamp() is not appropriate here... use `if (1 <= inten+i && inten+i <= 100) {...}` instead